.circleci/config.yml

```diff
@@ -9,7 +9,7 @@ jobs:
       - setup_remote_docker
       - run:
           name: Test
-          command: COMMIT_HASH=$(git rev-parse --short HEAD) hokusai test --build
+          command: COMMIT_HASH=$(git rev-parse HEAD) CODECOV_TOKEN=$CODECOV_TOKEN hokusai test --build
```

Comment:
It would be good to have these dynamic build args pulled in via Hokusai, and specified in a project's yml config - perhaps in a preprocessing step...

Reply:
I think, ideally, we wouldn't need to "thread" this Git-state information down to further Docker-layers. Maybe once we introduce this approach to another codebase, we refactor this to be a bit less coupled and a bit flatter.

I think we should re-visit this when we deem it worthwhile to measure code coverage across many projects though, and opting not to change now